main.go

```diff
@@ -36,6 +40,9 @@ func versionString() string {
 	} else {
 		version = fmt.Sprintf("%s-dev", Version) // no SHA. '0.x.y-dev' indicates it is run from source without build script.
 	}
+	if info, ok := debug.ReadBuildInfo(); !ok && info == nil {
+		version += " (non-modules)"
```

Comment:
Note that it's also possible to list all module versions if that's something you're interested in.

Reply:
We can add that later. I think this is good for now.